2.5/Dockerfile.fedora

```diff
@@ -0,0 +1,84 @@
+FROM registry.fedoraproject.org/f28/s2i-base:latest
+
+# This image provides a Ruby 2.5 environment you can use to run your Ruby
+# applications.
+
+EXPOSE 8080
+
+ENV NAME=ruby \
+    RUBY_VERSION=2.5 \
+    VERSION=0 \
+    RELEASE=1 \
+    ARCH=x86_64
+
+ENV SUMMARY="Platform for building and running Ruby $RUBY_VERSION applications" \
+    DESCRIPTION="Ruby $RUBY_VERSION available as container is a base platform for \
+building and running various Ruby $RUBY_VERSION applications and frameworks. \
+Ruby is the interpreted scripting language for quick and easy object-oriented programming. \
+It has many features to process text files and to do system management tasks (as in Perl). \
+It is simple, straight-forward, and extensible."
+
+LABEL summary="$SUMMARY" \
+      name="$FGC/$NAME" \
+      com.redhat.component="$NAME" \
+      release="RELEASE.$DISTTAG" \
```

Comment:
Please remove release label and RELEASE variable from all Dockerfile. It's no longer used and image release is incremented automatically by OSBS in every image build.